src/cljc/athens/common_events.cljc

```diff
@@ -347,8 +346,8 @@
 
 (defn build-unlinked-references-link-all
   "Builds `:datascript/unlinked-references-link` event with:
-  - `unlinked-refs`:  vector of blocks with unlinked refs
-  - `title        `: title of the page"
+  - `unlinked-refs`: list of maps that contains the str and id of unlinked refs
```

Comment:
"and id" or "and `:block/uid`"?

Reply:
It's `:block/uid`. I also updated `str` to `:block/string` to make it consistent.